vignettes/bisg.Rmd

```diff
@@ -69,88 +96,33 @@ Check the dimensions (the number of rows and columns) of the voter file.
 # Get the dimensions of the voter file
 dim(voter_file_geocoded)
 ```
-There are 14310 voters (or observations) and 14 columns in the voter file.
+There are 12 voters (or observations) and 25 columns in the voter file.
 
-Make sure to load your census data that details certain geographies (i.e. counties, cities, tracts, blocks, etc.)
-```{r}
-# Load New York census data
-load("~/shared/east_ramapo/data/ny_census2.RData")
-census_data <- ny_3
-```
-
-Next, load the state shape file using the sf::st_read() function.
+Convert geometry column name into two columns for latitude and longitude points.
 ```{r}
-# Load New York block shape file
-ny_shape <- st_read("~/shared/east_ramapo/data/tl_2014_36_tabblock10.shp")
+voter_file_geocoded <- voter_file_geocoded %>%
+  extract(geometry, c("lon", "lat"), "\\((.*), (.*)\\)", convert = TRUE)
 ```
 
-
 ### Step 2: De-duplicate the voter file.
 
-The next step involves removing duplicate voter IDs from the voter file, using the `dedupe_voter_file` function. Check the column name for the unique identifier assigned to each voter the voter file. For the East Ramapo voter file we are using, voter_id is the column named `id`.
-
+The next step involves removing duplicate voter IDs from the voter file, using the `dedupe_voter_file` function. 
 ```{r}
-# Remove duplicate voter IDs (the unique identifier for each voter)
-voter_file_geocoded <- dedupe_voter_file(voter_file = voter_file_geocoded, voter_id = "voter_id")
-
-# Check new dimensions of voter file after removing duplicate voter IDs
-dim(voter_file_geocoded)
+# Rename registration_id to voter_id
+names(voter_file_geocoded)[names(voter_file_geocoded) == "registration_number"] <- "voter_id"
```

Comment:
Since the `eiCompare` functions can accept a voter ID column name, it might be useful to demonstrate that (using `registration_number`) rather than changing the column name manually

Reply:
Thanks! noted.